.github/workflows/scale-test-egw.yaml

```diff
@@ -0,0 +1,485 @@
+
+          managedNodeGroups:
+          - name: ng-amd64-client
+            instanceTypes:
+            - m5.large
+            desiredCapacity: ${{ steps.vars.outputs.num_client_nodes }}
+            maxPodsPerNode: 100
+            spot: false
+            privateNetworking: true
+            volumeType: "gp3"
+            volumeSize: 10
+            ssh:
+              allow: true
+              publicKeyPath: ${{ env.AWS_SSH_KEY }}
+            taints:
+            - key: "node.cilium.io/agent-not-ready"
+              value: "true"
+              effect: "NoExecute"
+            labels:
+              role.scaffolding/egw-client: "true"
+          - name: ng-amd64-egw-node
+            instanceTypes:
+            - m5.large
+            desiredCapacity: 1
+            spot: false
+            privateNetworking: true
+            volumeType: "gp3"
+            volumeSize: 10
+            ssh:
+              allow: true
+              publicKeyPath: ${{ env.AWS_SSH_KEY }}
+            taints:
+            - key: "node.cilium.io/agent-not-ready"
+              value: "true"
+              effect: "NoExecute"
+            labels:
+              role.scaffolding/egw-node: "true"
+          - name: ng-amd64-heapster
+            instanceTypes:
+            - m5.large
+            desiredCapacity: 1
+            spot: false
+            privateNetworking: true
+            volumeType: "gp3"
+            volumeSize: 10
+            ssh:
+              allow: true
+              publicKeyPath: ${{ env.AWS_SSH_KEY }}
+            taints:
+            - key: "node.cilium.io/agent-not-ready"
+              value: "true"
+              effect: "NoExecute"
+            labels:
+              role.scaffolding/monitoring: "true"
+          - name: ng-amd64-no-cilium
+            instanceTypes:
+            - m5.large
+            desiredCapacity: 1
+            spot: false
+            privateNetworking: true
+            volumeType: "gp3"
+            volumeSize: 10
+            ssh:
+              allow: true
+              publicKeyPath: ${{ env.AWS_SSH_KEY }}
+            taints:
+            - key: "cilium.io/no-schedule"
+              value: "true"
+              effect: "NoSchedule"
+            labels:
+              cilium.io/no-schedule: "true"
+          EOF
+
+          eksctl create cluster -f ./eks-config.yaml
+
+      - name: Patch AWS-CNI to enable prefix delegation
+        run: |
+          kubectl set env daemonset aws-node -n kube-system ENABLE_PREFIX_DELEGATION=true
+          kubectl rollout restart ds/aws-node -n kube-system
+          kubectl rollout status ds/aws-node -n kube-system -w
+
+      - name: Install Cilium CLI
+        uses: cilium/cilium-cli@99ff3d9edd4050d7b5c9d1f51dcf6ebeeb613d15 # v0.16.15
+        with:
+          skip-build: ${{ env.CILIUM_CLI_SKIP_BUILD }}
+          image-repo: ${{ env.CILIUM_CLI_IMAGE_REPO }}
+          image-tag: ${{ env.CILIUM_CLI_VERSION }}
```

Comment:
I guess this should be:

```
image-tag: ${{ inputs.SHA || github.sha }}
```

after the cilium-cli repo merge.

Reply:
Yep definitely, thanks.